packages/cli/helpers/build.ts

```diff
@@ -33,20 +30,20 @@ const cliLifecyclePlugin: esbuild.Plugin = {
       })
 
       // we copy the contents from checkpoint-client to build
-      await copyFile(
+      await fs.promises.copyFile(
```

Comment:
We could import `fs` from `fs/promises` instead to avoid writing `fs.promises` everywhere, but either is fine with me.

Reply:
Yes! this PR prepares the move if we want to, I need to work on other things now.